### PR TITLE
adding new service manage-case-assignment under CDM for now

### PR DESCRIPTION
**Pull request by smathangi**
https://tools.hmcts.net/jira/browse/RDM-8368

### CODE REVIEW

File: jobdsl/organisations-beta.groovy [hunk 1]
```diff
@@ -11,7 +11,7 @@ List<Map> orgs = [
         [name: 'Adoption'],
         [name: 'AM'],
         [name: 'BSP', regex: '(send-letter-client|send-letter-service|send-letter-performance-tests|bulk-scan-.*|blob-router-service|reform-scan-.*)'],
-        [name: 'CDM', regex: 'ccd.*'],
+        [name: 'CDM', regex: '(ccd.*|manage-case-assignment)'],
```
Comment (timja) [inline jobdsl/organisations-beta.groovy:14]: why not prefix it with ccd?

Comment (smathangi) [reply jobdsl/organisations-beta.groovy:14]: I was told this will not be part of core CCD.

Comment (smathangi) [reply jobdsl/organisations-beta.groovy:14]: @mario-paniccia could you provide more details on it please ^^

Comment (mario-paniccia) [reply jobdsl/organisations-beta.groovy:14]: strictly speaking `manage-case-assignment` is not a micro part of the CCD application. So would not be correct prefixing it with ccd. But it's a micro implemented and maintained by the CDM team so in Jenkins I think it makes sense to have it under the CDM tab

Comment (timja) [reply jobdsl/organisations-beta.groovy:14]: who is maintaining it and what product does it belong under?

it should be prefixed with something grouping it to a product

Comment (mario-paniccia) [reply jobdsl/organisations-beta.groovy:14]: CDM team is the maintainer. 
We have been told the Product name should be aac for Assign Access To Case.
If prefixing with the product name is mandatory then we'll have to rename this to aac-manage-case-assignment.

Comment (timja) [reply jobdsl/organisations-beta.groovy:14]: yes it should be renamed

Comment (smathangi) [reply jobdsl/organisations-beta.groovy:14]: @timja  could you please confirm whether product prefix is mandatory for github repo?

Comment (smathangi) [reply jobdsl/organisations-beta.groovy]: @timja  is there anything else required in jenkins-config? Otherwise could you approve please.

**Pull request merged**